### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by RikuVirtanen**
First implementation with net6 targeting.

### CODE REVIEW

File: Frends.SFTP.ReadFile/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [1.0.0] - 2022-03-10
```
Comment (Svenskapojkarna) [inline Frends.SFTP.ReadFile/CHANGELOG.md:3]: Please update the date.

File: Frends.SFTP.ReadFile/Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.csproj [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+	  <Version>1.0.0</Version>
+	  <Authors>Frends</Authors>
+	  <Copyright>Frends</Copyright>
+	  <Company>Frends</Company>
+	  <Product>Frends</Product>
+	  <PackageTags>Frends</PackageTags>
+	  <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	  <Description>Executes file transfer through SFTP connection.</Description>
```
Comment (Svenskapojkarna) [inline Frends.SFTP.ReadFile/Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.csproj:18]: I think this should be something like read file or get file from SFTP server or something.

**Comment by Svenskapojkarna** [comment]
Please also add the link to ReadFile README to https://github.com/FrendsPlatform/Frends.SFTP/blob/main/README.md

File: Frends.SFTP.ReadFile/Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.csproj (hunk 2 above)
Comment (RikuVirtanen) [reply Frends.SFTP.ReadFile/Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.csproj:18]: Fixed with "Read file through SFTP connection."

**Pull request merged**